src/status_im/ui/screens/wallet/choose_recipient/views.cljs

```diff
@@ -45,7 +45,7 @@
    [react/touchable-highlight {:style    (styles/recipient-touchable true)
                                :on-press #(react/get-from-clipboard
                                             (fn [clipboard]
-                                              (re-frame/dispatch [:choose-recipient clipboard])))}
+                                              (re-frame/dispatch [:choose-recipient clipboard ""])))}
```

Comment:
`name` should be `nil`

Reply:
https://github.com/status-im/status-react/blob/918ab9caa7eae8c217671f8afdc6bd4e87bef85c/src/status_im/ui/screens/wallet/components/views.cljs#L62